GitCommands/CommitMessageManager.cs

```diff
@@ -36,22 +37,24 @@ public sealed class CommitMessageManager : ICommitMessageManager
         private readonly string _amendSaveStatePath;
         private readonly string _commitMessagePath;
         private readonly string _mergeMessagePath;
-
+        [CanBeNull]
+        private string _overriddenCommitMessage;
         private Encoding _commitEncoding;
         private IFileSystem _fileSystem;
```

Comment:
Nitpick: In new files like this, we use alphabetical order for class members of the same category.

Reply:
No problem. But I don't understand your request. Where do you want me to put it? its defined after "m" which is alphabetically ordered already.

Reply:
`private readonly` is one category `private [non-readonly]` another.
Each category should be sorted alphabetically.
I.e. move the declaration below `_fileSystem`.

Reply:
thanks